packages/psss/src/components/Table/TableRow.js

```diff
@@ -2,11 +2,11 @@
  * Tupaia
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
-import styled from 'styled-components';
-import { ExpandableTableRow } from '@tupaia/ui-components';
+import styled, { css } from 'styled-components';
+import { TableRow } from '@tupaia/ui-components';
 import * as COLORS from '../../constants/colors';
 
-export const SimpleTableRow = styled(ExpandableTableRow)`
+export const SimpleTableRowStyles = css`
```

Comment:
Nice style reuse here!!

I believe this is not used elsewhere?

```suggestion
const SimpleTableRowStyles = css`
```

Reply:
yeah I only recently found out that you can use styled-components to create css like this. pretty handy.